scripts/isct_test_function.py

```diff
@@ -433,6 +433,39 @@ def get_parser():
         print results_display.to_string()
         print 'Status: 0: Passed | 1: Crashed | 99: Failed | 200: Input file(s) missing | 201: Ground-truth file(s) missing'
 
+        if verbose == '2':
```

Comment:
I never came across the implementation of verbosity in tests. Ideally, it should be an command on its own such as `sct_visualize_tests` and load the stored data points into a file. It will make the code reusable in other situations.